ci/pipeline.yml

```diff
@@ -5,7 +5,7 @@ var:
     uri: https://github.com/instana/java-buildpack-memory-calculator.git
     username: ((project-berlin-gh-token))
     password: x-oauth-basic
-    branch: &build-branch ((branch))
```

Comment:
This is not very nice. We're creating and building pipelines for every version; https://github.com/instana/delivery-infra/blob/25c81ace81620ac5bc0bf3731179dd0160db21c4/pipelines.yml#L209
Now its creating a pipeline for every branch but then only building `main` which is confusing.
See our other projects, as they include ways to build, but not push a new version if current branch is not `main`.